backend-api/src/functions/asyncActiveSession/registeredLogs.ts

```diff
@@ -4,11 +4,21 @@ import {
 } from "../services/logging/commonRegisteredLogs";
 import { RegisteredLogMessages } from "../services/logging/types";
 
-export type MessageName = "AUTHENTICATION_HEADER_INVALID" | CommonMessageNames;
+export type MessageName =
+  | "AUTHENTICATION_HEADER_INVALID"
```

Comment:
This sounds like a common log in the making?

Reply:
Yeah agreed, we did discuss this. I would probably update this in a follow-up PR to this one so we could update this in the credential lambda at the same time

Reply:
yep absolutely.